filebeat/config/config.go

```diff
@@ -44,32 +44,32 @@ type FilebeatConfig struct {
 }
 
 type ProspectorConfig struct {
-	Paths                 []string
+	ExcludeFiles          []string `yaml:"exclude_files"`
+	ExcludeFilesRegexp    []*regexp.Regexp
```

Comment:
Yes, that is the case currently. Also see below with the duration. I agree that we should probably separate these in the future.